runelite-client/src/main/java/net/runelite/client/plugins/zoom/ZoomPlugin.java

```diff
@@ -84,7 +84,7 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 
 		if ("innerZoomLimit".equals(event.getEventName()) && zoomConfig.innerLimit())
 		{
-			intStack[intStackSize - 1] = 1200;
+			intStack[intStackSize - 1] = 1004;
```

Comment:
I would move this to a static final int with javadoc explaining why it is what it is